.github/workflows/quick.yaml

```
@@ -45,7 +45,10 @@ jobs:
     steps:
       - name: Install prerequisite packages
         run: |
+          # required for "apt-get build-dep" to work
+          sudo sed --in-place -E 's/# (deb-src.*updates main)/  \1/g' /etc/apt/sources.list
           sudo apt-get --quiet=2 update
+          sudo apt-get --quiet=2 build-dep squid
```

Please undo these out-of-scope changes.

```suggestion
          sudo apt-get --quiet=2 update
```